frontend/src/pages/Newsfeed.jsx

```
@@ -1,18 +1,65 @@
-    <div className="max-w-2xl mx-auto p-4">  
+    <div className="min-h-screen bg-gray-100">
+      {/* Main Container with Flexbox Layout */}
+      <div className="flex justify-between p-6">
+        {/* Left Sidebar: Create Event Form and Events Section */}
+        <aside className="w-1/3 bg-white shadow-md rounded-lg p-4 overflow-y-auto mt-20"> {/* Added mt-10 for top margin */}
+          {/* Dropdown or Button to Toggle Create Event Form */}
+          <button
+            onClick={toggleCreateEventForm}
+            className="w-full py-2 px-4 bg-blue-500 text-white rounded-lg mb-4 transition-all duration-300 hover:bg-blue-600"
+          >
+            {showCreateEventForm ? "Hide Create Event Form" : "Create New Event"}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)